articles/machine-learning/how-to-create-attach-kubernetes.md

```diff
@@ -424,6 +430,21 @@ Many webservice failures in AKS can be debugged by connecting to the cluster usi
 az aks get-credentials -g <rg> -n <aks cluster name>
 ```
 
+### Delete azureml-fe related resources
```

Comment:
Moved this guidance to be between the detach and reattach code sections (lines 396-398). We can delete lines 433-447

Reply:
> @mjaow I'm from the content team and made some suggestions to help with flow when incorporating the new guidance. Let me know what you think.

lgtm

Reply:
@majow great! To incorporate my suggestions, please click the "Commit suggestion" button and we can get this into the publishing queue.